src/tests/youtube/Cobalt_URL_002.test.js

```diff
@@ -0,0 +1,58 @@
+import {
+  setCobaltUrl,
+  pluginDeactivate,
+  pluginActivate,
+  getCobaltUrl,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'Cobalt URL - 002',
+  description: 'Loads valid URL and check the Cobalt behavior',
+  context: {
+    url: 'https://www.google.com/',
+  },
+  steps: [
+    {
+      description: 'Deactivate Cobalt browser',
+      test: pluginDeactivate,
+      params: constants.youTubePlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Activate Cobalt plugin',
+      test: pluginActivate,
+      params: constants.youTubePlugin,
+      assert: 'resumed',
+    },
+    {
+      description: 'Load URL and check the response',
+      sleep: 10,
+      test() {
+        return setCobaltUrl.call(this, this.$context.read('url'))
```

Comment:
Same here, suprised this is actually available. Does Cobalt really navigate to another URL?

Reply:
Yeah Wouter. This is working and it is navigating to google URL